resip/stack/ssl/DtlsTransport.cxx

```diff
@@ -167,9 +167,8 @@ DtlsTransport::_read( FdSet& fdset )
    // something about MSG_PEEK|MSG_TRUNC in Stevens..
    // .dlb. RFC3261 18.1.1 MUST accept 65K datagrams. would have to attempt to
    // adjust the UDP buffer as well...
-   unsigned int bufferLen = UdpTransport::MaxBufferSize + 5 ;
-   char* buffer = new char[ bufferLen ] ;
-   unsigned char *pt = new unsigned char[ bufferLen ] ;
+   std::unique_ptr<char[]> buffer(MsgHeaderScanner::allocateBuffer(MaxMessageSize));
```

Comment:
So we are allocating 2 64K buffers for every _read operation?  Might it make sense to make these class members and resuse them (like in UdpTransport)?  I see a copy operation below, so this is using the copy solution.